homepage/models.py

```diff
@@ -40,6 +40,65 @@ def get_details(self):
         return (self.course_id, self.name, self.mandatory, self.credit_points, self.syllabi,
                 self.avg_load, self.avg_rating, self.num_of_raters, self.num_of_reviewers)
 
+    # --- get all courses sorted by id/name/rating  - return QuerySets
+    @staticmethod
+    def get_all_courses_sorted_by_id():
+        return Course.objects.all()

```

Comment:
Missing here ```.order_by```, or is ordered by id by default?
Maybe worth a comment

Reply:
ordered by id as default, yes